### PR TITLE
How to Connect with the WildFly Community

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://github.com/wildfly/wildfly.org/issues/502

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@fjuma @bstansberry please review the article.

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title:  "How to Connect with the WildFly Community"
+date:   2024-05-31
+tags:   WildFly Community
+author: rchakrab
+description: Article about How to connect with the WildFly Community.
+---
+
+= How to Connect with the WildFly Community
+
+WildFly is an open-source, production-ready, flexible and lightweight application server, offers all the necessary features to run and deploy Java applications. Connecting with the WildFly community can provide valuable support, enhance your understanding of the server, and offer opportunities to contribute to the project. Here's how you can get involved:
```
Comment (bstansberry) [inline _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:12]: s/offers/offering

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+=== Fork the WildFly Repository on GitHub and Play with It
+Explore WildFly's https://github.com/wildfly/wildfly[source code] by https://github.com/wildfly/wildfly/fork[forking] the repository on GitHub. Clone your forked repository to your local machine, experiment with the code, and understand the inner workings of WildFly. This hands-on approach can deepen your knowledge and potentially lead to contributions to the project. You can also go through the article on https://www.wildfly.org/news/2024/03/01/How-to-contribute-to-WildFly/[How to contribute to WildFly].
+
+== Join the WildFly Google Group and Mailing List
+=== Engage in the WildFly Forums
+Participate in the https://groups.google.com/g/wildfly[WildFly Google Group] to ask questions, share knowledge, and engage with other users and developers. The forum is a valuable platform for seeking help, providing answers, and staying updated with announcements and community discussions.You can also chat with the WildFly community members as well as with contributors via our official https://wildfly.zulipchat.com/[Zulip] chatroom.
```
Comment (bstansberry) [inline _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:27]: s/discussion.You/discussions. You/g

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+=== Follow WildFly on Social Media
+Stay informed and connected by following WildFly on social media platforms. Follow us on https://x.com/WildFlyAS[X] and https://fosstodon.org/@wildflyas[Mastodon] for quick updates, announcements, and and subscribe to the WildFly https://www.youtube.com/@WildFlyAS[YouTube] channel for video tutorials, webinars, and conference talks.
```
Comment (bstansberry) [inline _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:30]: s/and and/and

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Participate in WildFly community events such as mini-conferences and webinars. These events offer learning opportunities through sessions and workshops by experts, networking with other users and developers, and community-building activities. Keep an eye on the WildFly website and social media channels for announcements about upcoming events.
+
+== Provide and Receive Support
+=== Ask for Help
+Don’t hesitate to seek support through forums, mailing lists, or social media channels. The WildFly community is active and responsive, making it a great resource for troubleshooting and advice.
+
+=== Offer Help
+Share your knowledge and expertise to help others in the community. By answering questions and providing guidance, you contribute to the collective knowledge base.
+
+=== Submit Code or contribute to Documentation
```
Comment (bstansberry) [inline _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:42]: s/contribute/Contribute

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+WildFly uses JIRA for issue tracking. You can find the list of issues and tasks on the https://issues.redhat.com/projects/WFLY/issues[WildFly JIRA dashboard]. If you've made improvements or fixed bugs in your forked repository, consider submitting pull requests to contribute back to the WildFly project. You can also visit WildFly official https://docs.wildfly.org/[documentation] and feel free to suggest improvements, or directly contributing updates and corrections. Your contributions can help improve WildFly for everyone.
```
Comment (bstansberry) [inline _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:43]: s/visit WildFly/visit the WildFly/g

File: _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc (hunk 2 above)
Comment (bstansberry) [reply _posts/2024-05-31-How-to-connect-with-WildFly-community.adoc:27]: I suggest adding something like:

"For discussions about using WildFly, try the https://wildfly.zulipchat.com/#narrow/stream/196266-wildfly-user[wildfly-user stream] in the chatroom.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks Ranabir. I've made some comments.

WDYT about adding a quick section about RSS? It's kind of old school but we publish a feed at https://www.wildfly.org/feed.xml that's updated whenever anything is  published under wildfly.org/news.

**Comment by RanabirChakraborty** [comment]
Thanks @bstansberry I have modified the article accordingly. Please review once.

**Comment by bstansberry** [comment]
Thanks @RanabirChakraborty!

**Pull request merged**